docs/Prowlarr/prowlarr-setup-limited-api.md

```diff
@@ -0,0 +1,51 @@
+# How to setup indexers with limited API
+
+So you have access to a few (unpaid) usenet indexers with limited API calls and you still want to add them as backup for failed downloads, but you're always hitting the API limits in the first hour and can't make fully use of them.
+
+In this Guide i will show you a trick how you can make more use of them without hitting the API limit in the first hour.
+
+!!! info "This is mainly a usenet option but might be useful for a few trackers that have rate limits. I will use for this guide mainly usenet indexers as examples."
+
+## Sync Profiles
+
+This is the same you see in the Starr apps under indexers when you open the specific indexer settings.
+
+A short description what they do, mean and when they are used.
```

Comment:
Would change to "A short description of what they do, what they mean, and when they are used."